dotcom-rendering/src/components/SetABTests.importable.tsx

```diff
@@ -120,9 +128,17 @@ export const SetABTests = ({
 			participations,
 		});
 
+		setBetaABTests(betaAb);
+
+		window.guardian.modules.abTests.isUserInTest = (
```

Comment:
Is `window.guardian.modules.abTests` defined anywhere?

Reply:
I've added it to the `window.guardian` model: dotcom-rendering/src/model/guardian.ts